commit b57ce68fae2ce2642a36746b8e7b7216a02bfc8c
Author: Peter Volkov <pva@gentoo.org>
Date:   Sun Feb 22 19:40:34 2009 +0300

    vz-gentoo.in typo: check for DST_CRONSCRIPT_DIR
    
    Fixed indentation and typo to make script really check for
    $DST_CRONSCRIPT_DIR directory existence.

diff --git a/etc/init.d/vz-gentoo.in b/etc/init.d/vz-gentoo.in
index 005813f..97ea313 100755
--- a/etc/init.d/vz-gentoo.in
+++ b/etc/init.d/vz-gentoo.in
@@ -56,17 +56,17 @@ check_config() {
 # being enabled or not.
 setup_cron()
 {
-       [ -z "$SRC_CRONSCRIPT_DIR" ] && return
-       [ -d "$SRC_CRONSCRIPT_DIR" ] || return
-       install -m644 -o root -g root $SRC_CRONSCRIPT_DIR/vz \
-               $DST_CRONSCRIPT_DIR
+	[ -z "$SRC_CRONSCRIPT_DIR" ] && return
+	[ -d "$DST_CRONSCRIPT_DIR" ] || return
+	install -m644 -o root -g root $SRC_CRONSCRIPT_DIR/vz \
+		$DST_CRONSCRIPT_DIR
 }
 
 remove_cron()
 {
-       [ -z "$SRC_CRONSCRIPT_DIR" ] && return
-       [ -d "$SRC_CRONSCRIPT_DIR" ] || return
-       cat > $DST_CRONSCRIPT_DIR/vz <<EOF
+	[ -z "$SRC_CRONSCRIPT_DIR" ] && return
+	[ -d "$DST_CRONSCRIPT_DIR" ] || return
+	cat > $DST_CRONSCRIPT_DIR/vz <<EOF
 # DO NOT EDIT THIS FILE!
 #
 # Contents of this file managed by /etc/init.d/vz script
